generative/3d_ldm/README.md

```diff
@@ -0,0 +1,134 @@
+# 3D Latent Diffusion Example
+This folder contains an example for training and validating a 3D Latent Diffusion Model on Brats data. The example includes support for multi-GPU training with distributed data parallelism based on a tutorial designed for a single GPU. https://github.com/Project-MONAI/GenerativeModels/blob/main/tutorials/generative/3d_ldm/3d_ldm_tutorial.ipynb
```

Comment:
Curious if the link "https://github.com/Project-MONAI/GenerativeModels/blob/main/tutorials/generative/3d_ldm/3d_ldm_tutorial.ipynb" mentioned here is for a purpose. Thanks

Reply:
The code is multi-gpu version adapted from this link. If users wants to run it on single gpu, this link may be enough for them.

Reply:
Would this be more clear to you? I changed it to:

The example includes support for multi-GPU training with distributed data parallelism based on a [tutorial designed for a single GPU](https://github.com/Project-MONAI/GenerativeModels/blob/main/tutorials/generative/3d_ldm/3d_ldm_tutorial.ipynb).